packages/next/client/index.js

```diff
@@ -30,6 +30,8 @@ if (!window.Promise) {
 const data = JSON.parse(document.getElementById('__NEXT_DATA__').textContent)
 window.__NEXT_DATA__ = data
 
+window.__NEXT_VERSION__ = { version: require('next/package.json').version }
```

Comment:
Okay as a global var? Or should I keep it within a random property and not make it as visible?

Reply:
`export const version = process.env.__NEXT_VERSION` should be enough!

Reply:
Nice, it is available!!

Updated